tests/test_fremdkostenblock.py

```diff
@@ -0,0 +1,92 @@
+        [
+            pytest.param(
+                Fremdkostenblock(
+                    kostenblockbezeichnung="teststring",
+                    kostenpositionen=[
+                        Fremdkostenposition(
+                            positionstitel="fremdkostenblocktitel",
+                            artikelbezeichnung="bsp",
+                            einzelpreis=Preis(
+                                wert=Decimal(3.50),
+                                einheit=Waehrungseinheit.EUR,
+                                bezugswert=Mengeneinheit.KWH,
+                                status=Preisstatus.ENDGUELTIG,
+                            ),
+                            betrag_kostenposition=Betrag(
+                                waehrung=Waehrungscode.EUR,
+                                wert=Decimal(12.5),
+                            ),
+                        )
+                    ],
+                    summe_kostenblock=Betrag(
+                        waehrung=Waehrungscode.EUR,
+                        wert=Decimal(98240),
+                    ),
+                ),
+                {
+                    "kostenblockbezeichnung": "teststring",
+                    "kostenpositionen": [
+                        {
+                            "marktpartnercode": None,
+                            "positionstitel": "fremdkostenblocktitel",
+                            "einzelpreis": {
+                                "wert": "3.5",
+                                "einheit": "EUR",
+                                "bezugswert": "KWH",
+                                "status": "ENDGUELTIG",
+                            },
+                            "bis": None,
+                            "menge": None,
+                            "zeitmenge": None,
+                            "artikelbezeichnung": "bsp",
+                            "marktpartnername": None,
+                            "artikeldetail": None,
+                            "von": None,
+                            "linkPreisblatt": None,
+                            "betragKostenposition": {"wert": "12.5", "waehrung": "EUR"},
+                            "gebietcodeEic": None,
+                        }
+                    ],
+                    "summeKostenblock": {"wert": "98240", "waehrung": "EUR"},
+                },
+                id="maximal attributes",
+            ),
+            pytest.param(
+                Fremdkostenblock(kostenblockbezeichnung="teststring"),
+                {
+                    "kostenblockbezeichnung": "teststring",
+                    "kostenpositionen": None,
+                    "summeKostenblock": None,
+                },
+                id="minimal attributes",
+            ),
+        ],
+    )
+    def test_serialization_roundtrip(self, fremdkostenblock, expected_json_dict):
+        """
+        Test de-/serialisation of Fremdkostenblock with minimal attributes.
```

Comment:
```suggestion
        Test de-/serialisation of Fremdkostenblock
```
du testest ja beides